src/main/java/com/github/theholywaffle/teamspeak3/TS3Api.java

```diff
@@ -2750,6 +2805,28 @@ public boolean moveClient(Client client, ChannelBase channel, String channelPass
 	}
 
 	/**
+	 * Moves a list of clients into the specified channel using the specified password.
+	 *
+	 * @param clients
+	 * 		the list of clients to move.
+	 * @param channel
+	 * 		the channel to move the client into
+	 * @param channelPassword
+	 * 		the password of the channel
+	 *
+	 * @return whether the command succeeded or not
+	 *
+	 * @querycommands 1
+	 */
+	public boolean moveClient(List<Client> clients, ChannelBase channel, String channelPassword) {
+		List<Integer> clientIds = null;
```

Comment:
List initialized to `null`, will cause NullPointerException. This should instead be `new ArrayList<>(clients.size())`.